### PR TITLE
feat(python): Change default number of rows printed in Notebooks for DataFrame/Series to 10

### PR DESCRIPTION
**Pull request by stinodego**
Closes https://github.com/pola-rs/polars/issues/14515

#### Changes
* For DataFrames: changed from 25 to 10.
* For Series: changed from 25 to ~20~ 10 also.
* Fix display when row limit is an odd number

~I guess we could also do 10 rows for both?~

### CODE REVIEW

File: py-polars/polars/dataframe/frame.py [hunk 1]
```diff
@@ -1819,18 +1819,19 @@ def _repr_html_(self, **kwargs: Any) -> str:
         """
         max_cols = int(os.environ.get("POLARS_FMT_MAX_COLS", default=75))
         if max_cols < 0:
-            max_cols = self.shape[1]
-        max_rows = int(os.environ.get("POLARS_FMT_MAX_ROWS", default=25))
+            max_cols = self.width
+
+        default_rows = 20 if _from_series else 10
+        max_rows = int(os.environ.get("POLARS_FMT_MAX_ROWS", default=default_rows))
```
Comment (stinodego) [inline py-polars/polars/dataframe/frame.py:1825]: This is the actual change, the rest are minor refactors.

**Comment by JulianCologne** [comment]
> I guess we could also do 10 rows for both?

imo 10 is nice and would be consistent.


Also might have found another formatting inconsistency: 🤔 
```python
df = pl.DataFrame({"a": pl.int_range(0, 100, eager=True)})
ser = df.get_column("a")
```

image

**Comment by stinodego** [comment]
> imo 10 is nice and would be consistent.

Let's go for that. Too much vertical scrolling in a Notebook is annoying.

File: py-polars/polars/dataframe/frame.py [hunk 2]
```diff
@@ -1819,18 +1819,18 @@ def _repr_html_(self, **kwargs: Any) -> str:
         """
         max_cols = int(os.environ.get("POLARS_FMT_MAX_COLS", default=75))
         if max_cols < 0:
-            max_cols = self.shape[1]
-        max_rows = int(os.environ.get("POLARS_FMT_MAX_ROWS", default=25))
+            max_cols = self.width
+
+        max_rows = int(os.environ.get("POLARS_FMT_MAX_ROWS", default=10))
```
Comment (stinodego) [inline py-polars/polars/dataframe/frame.py:1824]: Default was changed here from 25 to 10.

**Pull request merged**